src/test/java/org/prebid/server/auction/ExchangeServiceTest.java

```diff
@@ -3234,6 +3239,38 @@ public void shouldReturnBidsWithoutAdjustingPricesWhenAdjustmentFactorNotPresent
                 .containsExactly(BigDecimal.ONE);
     }
 
+    @Test
+    public void shouldReturnBidsAcceptedByPriceFloorEnforcer() {
+        // given
+        final BidderBid bid1 = givenBid(Bid.builder().id("bidId1").impid("impId1").price(ONE).build(), "USD");
```

Comment:
Usually we use firstBid. But in this situation should we highlight in bid name that bid should be enforced, like bidToEnforce or something similar?